tests/test_filesize.py

```diff
@@ -0,0 +1,15 @@
+import os
+from os.path import getsize
+
+
+dir = ('safebrowsing')
+filenames = []
+for file in os.listdir(dir):
+        if os.path.isfile(os.path.join(dir, file)) and 'mozstd-trackwhite' in file:
+            size = os.path.getsize(os.path.join(dir, file))
+            if size <= 200000:
+                filenames.append(file)
+            else:
+                print file + ' is over maximum filesize'
```

Comment:
Thanks for the tip! I'm not used to thinking about Python3 so I'll  make sure to change this.